core/src/test/java/org/apache/iceberg/TestTableMetadata.java

```diff
@@ -1235,6 +1247,47 @@ public void testUpdateSortOrder() {
         sortedByX.sortOrder().fields().get(0).nullOrder());
   }
 
+  @Test
+  public void testStatistics() {
+    Schema schema = new Schema(Types.NestedField.required(10, "x", Types.StringType.get()));
+
+    TableMetadata meta =
+        TableMetadata.newTableMetadata(
+            schema, PartitionSpec.unpartitioned(), null, ImmutableMap.of());
+    Assert.assertEquals(
+        "Should default to unsorted order", ImmutableList.of(), meta.statisticsFiles());
```

Comment:
Looks like this context string needs to be updated. This ins't testing sort order.